README.md

```diff
@@ -60,7 +60,18 @@ negotiation and thereby give rise to bitcoin's first truly risk less interest ra
 $ cargo update
 $ cargo build --release
 ```
-# Run
+
+# Running On Local Regtest Network (Recommended for developers)
```

Comment:
I don't think it's necessary to include "(Recommended for developers)", anyone doing development should be familiar with it.

Reply:
Thanks. Updated.